package.json

```diff
@@ -21,18 +24,22 @@
       "base/js/*"
     ]
   },
+  "resolutions": {
+    "tar": "4.4.2"
+  },
   "devDependencies": {
     "bootstrap-sass": "3.4.1",
     "extend": "^3.0.2",
     "gulp": "^4.0.0",
     "gulp-postcss": "^7.0.1",
     "gulp-replace": "^1.0.0",
-    "gulp-sass": "^4.0.1",
+    "gulp-sass": "^4.0.2",
     "gulp-sass-glob": "^1.0.9",
     "gulp-strip-css-comments": "^2.0.0",
     "gulp-transform-selectors": "0.0.2",
     "husky": "^1.0.0",
     "lint-staged": "^6.1.1",
+    "npm-force-resolutions": "0.0.2",
```

Comment:
Why do you use fixed version here and not in other places? I think we should have fixed versions everywhere. It is pretty common when a minor version update is not properly working.

Reply:
> Why do you use fixed version here and not in other places?

This line has been generated by 
```
npm i --save-dev npm-force-resolutions
```
(same for `gulp-transform-selectors`, which doesn't have a range)

>  I think we should have fixed versions everywhere

No need, that's what the `package-lock.json` file is for